cmd/notary-signer/main.go

```diff
@@ -191,7 +204,7 @@ func main() {
 		log.Println("HTTP server listening on", httpAddr)
 	}
 
-	err = server.ListenAndServeTLS(certFile, keyFile)
+	err = server.ListenAndServe()
```

Comment:
I’m afraid this doesn’t work: with this change the signer is serving over unencrypted HTTP instead of HTTPS.

(ListenAndServe has no equivalent of the tls.NewListener call in ListenAndServeTLS)